### PR TITLE
Update builtin data sources docs

### PR DESCRIPTION
**Pull request by Xantier**

### CODE REVIEW

File: content/docs/tech-insights/builtin-data-sources/index.md [hunk 1]
```diff
@@ -4,42 +4,53 @@ publishedDate: '2023-05-12T21:00:00.0Z'
...
+they currently include. You can get a list of the facts available on each data source within the app as well. You can
+learn how to access this information within the app on
+the [Data Source results and details page](../data-source-details/).
 
-This list will probably have better accuracy, since we update facts frequently. 
+This list will probably have better accuracy, since we update facts frequently.
 
-Please note, if you don't find what you are looking for and/or want to create something we haven't provided out of the box, Roadie Tech Insights lets you [define custom Data Sources](../define-custom-data-sources/).
+Please note, if you don't find what you are looking for and/or want to create something we haven't provided out of the
+box, Roadie Tech Insights lets you [define custom Data Sources](../define-custom-data-sources/).
```
Comment (punkle) [inline content/docs/tech-insights/builtin-data-sources/index.md:29]: This is the second time this is mentioned. Maybe say "As mentioned above.."

**Comment by punkle** [comment]
Is this just formatting or is there actual changes to the doc.

**Review by punkle — APPROVED** [review]

**Comment by Xantier** [comment]
> Is this just formatting or is there actual changes to the doc.

Both GitHub ones have been updated/added. There were some unneeded auth blocks that were removed

File: content/docs/tech-insights/builtin-data-sources/index.md [hunk 2]
```diff
@@ -4,42 +4,53 @@ publishedDate: '2023-05-12T21:00:00.0Z'
...
+We have built several Data Sources, which you can find explained below, together with annotations used by them and facts
+they currently include. You can get a list of the facts available on each data source within the app as well. You can
+learn how to access this information within the app on
+the [Data Source results and details page](../data-source-details/).
 
-This list will probably have better accuracy, since we update facts frequently. 
+This list will probably have better accuracy, since we update facts frequently.
 
-Please note, if you don't find what you are looking for and/or want to create something we haven't provided out of the box, Roadie Tech Insights lets you [define custom Data Sources](../define-custom-data-sources/).
+Please note, if you don't find what you are looking for and/or want to create something we haven't provided out of the
```
Comment (Xantier) [inline content/docs/tech-insights/builtin-data-sources/index.md:28]:
```suggestion
As mentioned above, if you don't find what you are looking for and/or want to create something we haven't provided out of the
```

**Pull request merged**